### DISCUSSION TITLE
Question: is there plugins?

### DISCUSSION THREAD

**Discussion by Mrs-Feathers** [misskey-dev/misskey#14417] [Q&A] [answered]
i just want to know if there are plugins for this, as i really like mastodon, but want to add my own features with plugins. like XMPP chat, a game arcade (litterally just pages with iframes or html5 code added to the side menu) and other small fun stuff that doesnt effect the core or functionality of mastodon. just small QoL plugins.

**Comment by KisaragiEffective** [comment, marked as answer]
Misskey does not have such mechanisms to add features without  modifying code base as far as I know.
